trainer/pyproject.toml

```diff
@@ -2,23 +2,18 @@
 name = "trainer"
 version = "0.1.0"
 description = ""
-authors = ["David Corbitt <davidlcorbitt@gmail.com>"]
```

Comment:
Shots fired

Reply:
Lol sorry that actually wasn't intentional, at one point in this PR I was moving away from poetry and onto pip entirely so I deleted all the poetry-generated config and then ended up adding it back in. Added back. 😂